assets/src/components/mapPage.tsx

```diff
@@ -243,9 +250,6 @@ const MapPage = (): ReactElement<HTMLDivElement> => {
         case SelectedEntityType.RoutePattern:
           fullStoryEvent("RPC Opened", {})
       }
-      if (selectedEntity) {
```

Comment:
@lemald @firestack I want to highlight this change, since it's different from what you approved. 

I moved the call to `setSearchOpen(true)` outside of `setVehicleSelection`, since on mobile some calls to `setVehicleSelection` (clicking on a vehicle from the map) should open the drawer, and others (clicking on a vehicle search result) shouldn't.

Reply:
Another note: The collection of use cases has gotten complicated enough that I will feel a lot more comfortable merging this once I've added better test coverage.